examples/node/register-contract.js

```diff
@@ -0,0 +1,32 @@
+const DashJS = require('dash');
+
+const sdkOpts = {
+  network: 'testnet',
+  mnemonic:'your mnemonic here'
+};
+const sdk = new DashJS.Client(sdkOpts);
+
+const registerContract = async function () {
+  await sdk.isReady();
+  let platform = sdk.platform;
+  const identity = await platform.identities.get('your identity id here');
+
+  const contractDocuments = {
+    note: {
+      properties: {
+        message: {
+          type: "string"
+        }
+      },
+      indices: [
+        { "message": "asc"}
+      ],
+      additionalProperties: false
+    }};
+
+  const contract = await platform.contracts.create(contractDocuments, identity);
+  await platform.dpp.dataContract.validate(contract)
```

Comment:
Looks confusing:
```js
platform.contracts.create(contractDocuments, identity);
platform.dpp.dataContract.validate(contract)
```

Reply:
In StateTransitionBuilder, we have a check on structure `await this.dpp.stateTransition.validateStructure(stateTransition)` 
If the stateTransition contains a contract, does it execute dataContract.validate ?
Or should we pass a `[records].validate()` methods ?

Reply:
Yes, when we validate ST we validate a contract inside too, but I would provide the validate data contract method (such as document and identity) for users in order to allow them to validate data before they send it (integrate to forms, handle errors, ...)